src/Table/index.tsx

```diff
@@ -131,48 +135,57 @@ export const Table = <TData extends object>({
                   {headerGroup.headers.map((header) => {
                     const canSort = enableSorting && header.column.getCanSort();
                     const isCustomHeader = header.column.columnDef.meta?.customHeader;
+                    const tableHeaderProps = {
+                      key: header.id,
+                      colSpan: header.colSpan,
+                      width: header.getSize(),
+                      sorted: header.column.getIsSorted(),
+                      canSort,
+                    };
+
                     const headerContents = header.isPlaceholder
                       ? null
-                      : flexRender(header.column.columnDef.header, header.getContext());
+                      : flexRender(header.column.columnDef.header, {
+                          ...tableHeaderProps,
+                          ...header.getContext(),
+                        });
 
                     const {
                       activeTab = '',
                       handleTabs = () => {},
                       tabs = [],
                     } = header.column.columnDef.meta?.columnTabs || {};
 
-                    return (
-                      <TableHeader
-                        key={header.id}
-                        colSpan={header.colSpan}
-                        width={header.getSize()}
-                        sorted={header.column.getIsSorted()}
-                        canSort={canSort}
-                      >
-                        {!!tabs.length && (
-                          <TableTabs activeTab={activeTab} handleTabs={handleTabs} tabs={tabs} />
-                        )}
-                        <SortButton
-                          canSort={canSort}
-                          onClick={header.column.getToggleSortingHandler()}
-                        >
-                          {isCustomHeader ? (
-                            headerContents
-                          ) : (
-                            <TableHeaderWrapper>{headerContents}</TableHeaderWrapper>
+                    if (isCustomHeader) {
+                      return headerContents;
+                    } else {
+                      return (
+                        <TableHeader {...tableHeaderProps}>
+                          {!!tabs.length && (
+                            <TableTabs activeTab={activeTab} handleTabs={handleTabs} tabs={tabs} />
+                          )}
+                          <SortButton
+                            canSort={canSort}
+                            onClick={header.column.getToggleSortingHandler()}
+                          >
+                            {isCustomHeader ? (
```

Comment:
Will this ever be `true`? Asking b/c of return on line 160

Reply:
nice catch, looks like this will always be falsey.

Reply:
not sure I follow this thread, but we were talking about that `meta` property in a different PR, coincidentally.
https://github.com/icgc-argo/rdpc-ui/pull/147/files#diff-a68ddbc20eef704019906015d66875a95925225e4642a3a4b630cb9b6399c13dR740

That line there is the one that would give a truthy value to `isCustomHeader` here (per [133/137](https://github.com/icgc-argo/uikit/pull/141/files/3a2948844c90efa9aca81a073097664d75b3c01a#diff-7557138a062a078c4b84616d3820ef6dfe3ae3b9b47904be66ab80b55dd64cbcR137)).

What am I missing for what you guys are agreeing on?